### ISSUE TITLE
collaborate with rcdimple?

### ISSUE THREAD

**Issue by timelyportfolio** [Bart6114/dimple#1]
Nice job.  I have been working on `dimple` with `rCharts` and then [`rcdimple`](https://github.com/timelyportfolio/rcdimple) for the last year and a half.  I would love to collaborate to have one solid `dimple` implementation but understand if you want your own package.

**Comment by bart6114** [comment]
@timelyportfolio, I completely agree; one solid implementation is preferable. I wasn't aware of the `rcdimple` package. Nice work, I like the integrated piping functionality. I'll take a closer look at `rcdimple` in the next couple of days.

**Comment by happyshows** [comment]
@timelyportfolio  @Bart6114  good work for both of you. I have been using ggvis and rcdimple for majority of the time. I love dimple chart allow user to put multiple categorical values (in groups) on x axis. I'd suggest Bart to start with addressing the issues with rcdimple package, where I submitted three tickets of course :p.
